### PR TITLE
[#295] Add metamagic gem

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**
Related to: https://github.com/fs/rails-base/issues/295

### CODE REVIEW

**Comment by ruslankhaertdinov** [comment]
@timurvafin please review

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -6,7 +6,10 @@ html class='no-js' lang='en'
     meta charset='utf-8'
```
Comment (timurvafin) [inline app/views/layouts/application.html.slim]: Would you like to move all meta tags to `metamagic`?

File: app/views/users/registrations/new.html.slim [hunk 2]
```diff
@@ -1,8 +1,4 @@
 .row
-  .columns
-    h2 = title
-
-.row
```
Comment (timurvafin) [inline app/views/users/registrations/new.html.slim]: Why this block was removed?

**Comment by timurvafin** [comment]
Probably we could remove `config/locales/titles`

**Comment by timurvafin** [comment]
Checkout https://github.com/fs/rails-base/blob/master/app/views/users/registrations/new.html.slim#L14
We could remove `t` from here.

File: app/views/users/registrations/new.html.slim (hunk 2 above)
Comment (ruslankhaertdinov) [inline app/views/users/registrations/new.html.slim]: B/c in this case title is blank. What do you think about this?

File: app/views/layouts/application.html.slim (hunk 1 above)
Comment (ruslankhaertdinov) [inline app/views/layouts/application.html.slim]: Ok

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -6,7 +6,10 @@ html class='no-js' lang='en'
...
     meta name='viewport' content='width=device-width, initial-scale=1.0'
 
-    title= title
+    = metamagic site: "RailsBase",
+                title: [:title, :site],
+                separator: " — ",
```
Comment (timurvafin) [inline app/views/layouts/application.html.slim]: Please use short dash here as separator: ` - `

**Comment by ruslankhaertdinov** [comment]
2015-07-30 14-16-36 railsbase

**Comment by ruslankhaertdinov** [comment]
@timurvafin Is this ok? https://github.com/fs/rails-base/pull/306/files#diff-2298ed2d82756e554b223be36080fa6fL2

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,8 +3,9 @@
 ## Unreleased
 - Add gem [rack-canonical-host](https://github.com/tylerhunt/rack-canonical-host) for the definition a single host name as the canonical host for your application.
 - Add Google Analytics
-
 - Add ability to close app from unauthorized viewers, see: https://github.com/fs/rails-base/pull/302/files
+- Replace [title](https://github.com/calebthompson/title) gem with [metamagic](https://github.com/lassebunk/metamagic)
+- Remove titles.en.yaml
```
Comment (timurvafin) [inline CHANGELOG.md]: Don't thing we need to mention this here. It's too verbose.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done.

File: Gemfile [hunk 5]
```diff
@@ -20,7 +20,7 @@ gem "uglifier", ">= 1.3.0"
 gem "active_link_to"
 gem "simple_form"
 gem "slim"
-gem "title"
+gem "metamagic"
```
Comment (timurvafin) [inline Gemfile]: Please order gems alphabetically in this section.

**Comment by timurvafin** [comment]
@ruslankhaertdinov looks good, except small issue. Please fix it and cleanup commits to be ready to merge.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

**Comment by timurvafin** [comment]
Merged

**Pull request closed without merging**